conda/recipes/nvstrings/meta.yaml

```
@@ -26,6 +26,7 @@ build:
 requirements:
   build:
     - cmake
+    - python
```

By default the version here won't be the same as in `host`. So we might need to do something like this...

```suggestion
    - python 3.6. # [py36]
    - python 3.7. # [py37]
```

---

Would specifying `python x.x` work for this?

---

Maybe? I'm not sure. That's sort of slowly getting phased out, but it's worth trying.

If not, this is a reasonable fallback. `x.x` should still work in `host` and `run` at least.

---

It does not work in build from the looks of it, will try your method